src/transformers/image_utils.py

```diff
@@ -96,6 +96,17 @@ def is_batched(img):
     return False
 
 
+def _is_scaled_image(image: np.ndarray) -> bool:
```

Comment:
It looks that all other functions in `image_utils.py` do not start with a leading underscore. I understand that it might be a hint to describe that this function is used internally only. 

Other functions like `is_batched` (as in [here](https://github.com/huggingface/transformers/blob/68fa9a5937ae7aa707f5ff2639aa36a37a0a9928/src/transformers/models/deta/image_processing_deta.py#L42) and [here](https://github.com/huggingface/transformers/blob/68fa9a5937ae7aa707f5ff2639aa36a37a0a9928/src/transformers/models/deta/image_processing_deta.py#L42)) are similarly used by other modules and do not have a leading underscore.

Do you think it makes sense to name it as `is_scaled_image`?